web/src/routes/+layout.svelte

```diff
@@ -218,6 +251,24 @@
 		color: var(--surface-foreground);
 		padding: 1rem;
 		border-radius: 0.5rem;
-		border: 2px solid var(--default-border);
+		border: var(--default-border);
+	}
+
+	:global(a) {
+		color: var(--accent-gray);
+	}
+
+	:global(a[target='_blank']) {
```

Comment:
@ やハッシュタグなどクリックできるものはすべて対象にしたいので main の中のやつ全部を対象にした方がいいかも？
```suggestion
	:global(main a) {
```